Automattic/wp-calypso
# Copy Site: add celebratory banner 'NOTICE_CELEBRATE_SITE_COPY' to home

#### Proposed Changes

* Display a new celebratory banner after copying a site and landing on `/home/{SITE_ID}`

#### Testing Instructions



* Apply D99397-code to your sandbox.
- Go to /sites.
- Open the actions menu on a business site.
- Click on Copy Site.
- Follow the normal copy site process through the checkout.
- When you land on the site home after finishing the copy site process, observe the new notice in the UI.
- Click on `Show site setup`
- Observe the site setup.

#### Screencast

https://user-images.githubusercontent.com/779993/214860128-e73ae138-0f18-4ace-bc79-6a3e81c17d66.mp4




#### Pre-merge Checklist





Related to:
- D99397-code
- Closes: https://github.com/Automattic/dotcom-forge/issues/1545
